vsutil.py

```diff
@@ -146,7 +151,7 @@ def frame2clip(frame: vs.VideoFrame, *, enforce_cache=True) -> vs.VideoNode:
     # Forcefully add a cache to Modify-Frame if caching is disabled on the core.
     # This will ensure that the filter will not include GIL characteristics.
     if not core.add_cache and enforce_cache:
-        result = result.std.Cache(size=1, fixed=True)
+        return result.std.Cache(size=1, fixed=True)
```

Comment:
I don't like this change:

If you want to add a feature afterwards, you need to remove the `return` and replace it back with `result = `

I suggest keeping the `result = `